tests/Unit/CrudPanel/CrudPanelColumnsTest.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+class CrudPanelColumnsTest extends BaseCrudPanelTest
+{
+    private $oneColumnArray = [
```

Comment:
The specification of the `oneBLAH` `twoBLAH` things here kind of make me think this could be done programatically and that whatever this is trying to test could be done within a loop with a bunch of expectations...

Reply:
This is a fairly common practice in unit testing, though. Sure, here you can probably do it, but not when it comes to testing more complex structures. And we'll get to that soon, when we test various create/update scenarios, so it's probably a good idea to have things consistent.

Reply:
Ok.